website/docs/intro.mdx

```diff
@@ -27,7 +27,7 @@ Some of the things Taqueria can do for you include:
 - Compiling smart contracts using the [LIGO](/docs/plugins/plugin-ligo/), [SmartPY](/docs/plugins/plugin-smartpy/), or [Archetype](/docs/plugins/plugin-archetype/) plugins
 - Running a local sandbox with the [Flextesa plugin](/docs/plugins/plugin-flextesa/)
 - Originating a smart contract to a sandbox and testnet with the [Taquito plugin](/docs/plugins/plugin-taquito/)
-- Generating TypeScript types from a Michelson contract with the [Contract Types plugin](/docs/plugins/plugin-typescript-generator/), providing Taquito type safety and autocompletion in VS Code
+- Generating TypeScript types from a Michelson contract with the [Contract Types plugin](/docs/plugins/plugin-contract-types/), providing Taquito type safety and autocompletion in VS Code
```

Comment:
This reads like Taquito type safety is a single concept. Does it make sense to word this as 'type safety for use with Taquito'?

Reply:
The additional context doesn't fit the pattern of the list anyway so removing